Cargo.toml

```diff
@@ -15,28 +15,51 @@ edition = "2018"
 all-features = true
 
 [dependencies]
+# These two are generally really common simple dependencies so it does not seem
+# much of a point to optimize these, but these could in theory be removed for
+# an indirection through std::Formatter.
 dtoa = "0.4"
 itoa = "0.4.3"
+
+# This is a dependency that already exists in url
 percent-encoding = "2.1"
-sha1 = ">= 0.2, < 0.7"
+
+# We need this for redis url parsing
 url = "2.1"
-combine = "3.8.1"
+
+# We need this for script support
+sha1 = { version = ">= 0.2, < 0.7", optional = true }
```

Comment:
I'm actually not so worried about that one self-contained dependency, but given it's default-enabled I also don't care too much about it.